src/providers/toast/toast.ts

```diff
@@ -34,32 +35,42 @@ export class ToastProvider {
     translateService.use('en');
   }
 
-  error(message: string, hideDelay?: number, position?: string): void {
+  error(message: string | TranslatableObject, hideDelay?: number, position?: string): void {
     this.show(message, this.TypeEnum.ERROR, hideDelay, position);
   }
 
-  success(message: string, hideDelay?: number, position?: string): void {
+  success(message: string | TranslatableObject, hideDelay?: number, position?: string): void {
     this.show(message, this.TypeEnum.SUCCESS, hideDelay, position);
   }
 
-  warn(message: string, hideDelay?: number, position?: string): void {
+  warn(message: string | TranslatableObject, hideDelay?: number, position?: string): void {
     this.show(message, this.TypeEnum.WARN, hideDelay, position);
   }
 
-  log(message: string, hideDelay?: number, position?: string): void {
+  log(message: string | TranslatableObject, hideDelay?: number, position?: string): void {
     this.show(message, this.TypeEnum.LOG, hideDelay, position);
   }
 
-  debug(message: string, hideDelay?: number, position?: string): void {
+  debug(message: string | TranslatableObject, hideDelay?: number, position?: string): void {
     this.show(message, this.TypeEnum.DEBUG, hideDelay, position);
   }
 
-  show(message: string, type?: number, hideDelay?: number, position?: string) {
+  show(messageOrObj: string | TranslatableObject, type?: number, hideDelay?: number, position?: string) {
     let cssClass = 'toast-service';
     if (this.TypeName[type]) {
       cssClass += ' toast-' + this.TypeName[type]
     }
-    this.translateService.get(message).subscribe((translation) => {
+    let message: string;
+    let parameters: any;
+    if (typeof messageOrObj === 'string') {
+      message = messageOrObj as string;
+      parameters = null;
+    } else {
+      const obj = messageOrObj as TranslatableObject;
+      message = messageOrObj.key;
```

Comment:
`obj.key` instead?

Reply:
already doing that, you're somehow reviewing on an older commit